packages/components/src/components/Dialog/styles/dialogStyles.ts

```diff
@@ -4,8 +4,8 @@ import { sv } from '#/utilities/styleUtilities';
 
 export const dialogContentStyles = sv({
     base:
-        'tw-group tw-block tw-fixed tw-left-2 tw-right-2 tw-top-1/2 sm:tw-left-1/2 sm:tw-transform sm:tw--translate-x-1/2 tw--translate-y-1/2 tw-bg-base tw-shadow-mid tw-z-20 ' +
-        'sm:tw-min-w-[--dialog-min-width] tw-max-w-[--dialog-max-width] tw-min-h-[--dialog-min-height] tw-max-h-[600px] tw-h-fit tw-overflow-hidden ' +
+        'tw-group tw-block tw-fixed tw-left-2 tw-right-2 tw-top-1/2 sm:tw-left-1/2 sm:tw-transform sm:tw--translate-x-1/2 tw--translate-y-1/2 tw-bg-base tw-shadow-mid tw-border tw-border-line tw-z-20 ' +
+        'sm:tw-w-fit sm:tw-min-w-[--dialog-min-width] tw-max-w-[--dialog-max-width] tw-min-h-[--dialog-min-height] tw-max-h-screen tw-h-fit tw-overflow-hidden ' +
```

Comment:
So it is fine on safari ios
```suggestion
        'sm:tw-w-fit sm:tw-min-w-[--dialog-min-width] tw-max-w-[--dialog-max-width] tw-min-h-[--dialog-min-height] tw-max-h-dvh tw-h-fit tw-overflow-hidden ' +
```